### PR TITLE
Add Security & Privacy Questionnaire

### PR DESCRIPTION
**Pull request by ayuishii**
Adds [security & privacy self-assessment](https://www.w3.org/TR/security-privacy-questionnaire/) for Storage Buckets API.

### CODE REVIEW

**Comment by ayuishii** [comment]
Friendly ping...

File: security-privacy-self-assessment.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+No.
+
+### 2.12. What temporary identifiers might this specification create or expose to the web?
+None.
+
+### 2.13. How does this specification distinguish between behavior in first-party and third-party contexts? 
+No distinction. This feature is respects a same-orgin policy and is inaccessible by third-party contexts. 
+
+### 2.14. How does this specification work in the context of a user agent’s Private Browsing or "incognito" mode?
+The behavior under Private Browser or "incognito" will remain unchanged to that of [IndexedDB](https://w3c.github.io/IndexedDB/) and [CacheStorage](https://w3c.github.io/ServiceWorker/#cachestorage) and will be inaccessible once incognito session is closed.
```
Comment (pwnall) [inline security-privacy-self-assessment.md:46]: User agents are expected to maintain separate state for Private Browsing sessions.

More concretely, the collection of buckets exposed to origins in a Private Browsing session should be separate from the collection exposed in a normal browsing session.

At the time of this writing, major browsers have RAM-backed implementations of storage APIs such as [IndexedDB](https://w3c.github.io/IndexedDB/) and [CacheStorage](https://w3c.github.io/ServiceWorker/#cachestorage) for Private Browsing sessions. We expect that these browsers will have RAM-backed implementations of Storage Buckets as well.

File: security-privacy-self-assessment.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+None.
+
+### 2.13. How does this specification distinguish between behavior in first-party and third-party contexts? 
+No distinction. This feature is respects a same-orgin policy and is inaccessible by third-party contexts. 
+
+### 2.14. How does this specification work in the context of a user agent’s Private Browsing or "incognito" mode?
+The behavior under Private Browser or "incognito" will remain unchanged to that of [IndexedDB](https://w3c.github.io/IndexedDB/) and [CacheStorage](https://w3c.github.io/ServiceWorker/#cachestorage) and will be inaccessible once incognito session is closed. 
+
+### 2.15. Does this specification have a "Security Considerations" and "Privacy Considerations" section?
+The specification will expect to have additions to security and privacy considerations about browser eviction.
```
Comment (pwnall) [inline security-privacy-self-assessment.md:49]: This explainer targets [the Storage specification](https://storage.spec.whatwg.org/). We expect to add a "Security Considerations" section discussing quota accounting (the `estimate()` API) and opaque responses, and a "Privacy Considerations" section discussing bucket eviction.

**Review by pwnall — CHANGES_REQUESTED** [review]
This is a very good first cut! Thank you, and apologies for the late review! I somehow missed this in my inbox.

I proposed some changes below. Please feel free to rephrase. Let's discuss if anything seems confusing / wrong.

**Comment by ayuishii** [comment]
Thanks for the review & corrections, I think this all makes sense to me.

**Review by pwnall — APPROVED** [review]
LGTM.  Thank you for the quick revisions!

**Pull request merged**